GiniVision/Classes/CameraViewController.swift

```diff
@@ -131,13 +143,15 @@ public typealias CameraErrorBlock = (_ error: CameraError) -> ()
         if let validCamera = camera {
             cameraState = .valid
             previewView.session = validCamera.session
-            (previewView.layer as! AVCaptureVideoPreviewLayer).videoGravity = AVLayerVideoGravityResizeAspect
+            (previewView.layer as! AVCaptureVideoPreviewLayer).videoGravity = AVLayerVideoGravityResizeAspectFill
```

Comment:
I think so, because otherwise the previewView shows black spaces on the sides when it is on portrait mode, and it does not look very good. The image taken is good and the extractions are the same.